internal/cmd/fetcher/main.go

```diff
@@ -128,11 +132,9 @@ func recreateNPMPackageLock(plugin createdPlugin) error {
 }
 
 // runPluginTests runs 'make test PLUGINS="org/name:v<new>"' in order to generate plugin.sum files.
-func runPluginTests(plugins []createdPlugin, rootDir string) error {
-	pluginsEnv := make([]string, 0, len(plugins)*2)
-	for _, plugin := range plugins {
-		pluginsEnv = append(pluginsEnv, fmt.Sprintf("%s/%s:%s", plugin.org, plugin.name, plugin.newVersion))
-	}
+func runPluginTests(plugin createdPlugin) error {
```

Comment:
It would be better to have this execute tests on all plugins - otherwise we end up running common tests (unrelated to a specific plugin) potentially way more than necessary. What's the thinking behind making it run on an individual plugin?

Reply:
I was under the impression we were running only tests required for testing / generating the hash file for a given plugin.

But this appears to be incorrect, we're calling `make test` which is go test./...

I don't get why we're running the entire test suite from within the fetcher, and not scoping it down to `go test ./tests `

Reply:
I'll change this to that slice though, as the point wrt to parallel testing is a good suggestion.